### ISSUE TITLE
Standard import "from user import X" should be placed before "from pylint import sys" (wrong-import-order)

### ISSUE THREAD

**Issue by camilonova** [PyCQA/pylint#2532]
### Steps to reproduce
1. Run this shell commands:
```
virtualenv --python=python3 foo
cd foo
source bin/activate
pip install pylint astroid --pre -U
mkdir user
touch user/__init__.py
echo "def hello():\n    return 'hi'" > user/models.py
echo "from pylint import sys\nfrom user.models import hello\n\ndef main():\n    print(sys.version, hello())" > main.py
pylint main.py
```
2. Output from pylint command is:
```
No config file found, using default configuration
************* Module main
C:  1, 0: Missing module docstring (missing-docstring)
C:  4, 0: Missing function docstring (missing-docstring)
C:  2, 0: standard import "from user.models import hello" should be placed before "from pylint import sys" (wrong-import-order)

------------------------------------------------------------------
Your code has been rated at 2.50/10 (previous run: 0.00/10, +2.50)
```

### Current behavior

Pylint is reporting a wrong-import-order on line 2

### Expected behavior

Should not report a `wrong-import-order` since `user` is not a python package in Python3

### pylint --version output
```
$ pylint --version
No config file found, using default configuration
pylint 1.8.4,
astroid 1.6.3
Python 2.7.10 (default, Oct  6 2017, 22:29:07)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]

$ pip --version
pip 18.0 from /private/tmp/foo/lib/python3.7/site-packages/pip (python 3.7)

$ python --version
Python 3.7.0
```

It seems that pylint is not using Python3 as it should be.

**Comment by camilonova** [comment]
On visual studio code or sublime text, in a django project:

![screen shot 2018-09-29 at 5 22 21 pm](https://user-images.githubusercontent.com/202559/46487191-0ddef380-c7c5-11e8-9b31-de3693294dd0.png)

**Comment by camilonova** [comment]
Reported long back at https://github.com/PyCQA/pylint/issues/1716

**Comment by camilonova** [comment]
@PCManticore can you please take a look?

**Comment by PCManticore** [comment]
@camilonova Thanks for creating an issue and thanks for waiting. In your example, `user` is a package but a namespace package, so you can still operate with it as if it was a normal package. Unfortunately the error you are seeing comes from `isort`. We use `isort` to figure out what kind of module (third party, local, stdlib) a module is, and in this case it suggests that it is a `stdlib` module. There's not much we can do here, I suggest opening an issue on `isort` side.

**Comment by camilonova** [comment]
@PCManticore Thank you, I guessed `isort` was involved back then so I opened an issue there https://github.com/timothycrosley/isort/issues/778

There is something else we can to to make this happen?
